### ISSUE TITLE
Add taxon restriction to "entry into host"

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#20757]
GO:0044409 entry into host

please add "never in vertebrates"
and "never in fly taxon"

This term is to annotate symbionts (virus, bacteria, yeast, oomycete, parasitic worm)

There are 1780 vertebrate annotations, 173 EXP
31 drosophila

this isn't evolved process

**Comment by balhoff** [comment]
This seems applicable to some vertebrates, e.g. https://en.wikipedia.org/wiki/Pearlfish

**Comment by ValWood** [comment]
Yes, but here the whole organism lives inside something. I don't think it would apply to gene-specific annotation would it?  The fish swims into the host, but swimming isn't the property of any specific gene product. Its a property of an entire organism.

**Comment by pgaudet** [comment]
Likewise for worms ? Maybe genes help enter into host **cells**? This would be consistent with children term: 

-  GO:0085017 entry into host cell by a symbiont-containing vacuole
 - GO:0044410 entry into host through natural portals
-  GO:0035635 entry of bacterium into host cell
-  GO:0075015 formation of infection structure
- GO:0046718 viral entry into host cell

**Comment by ValWood** [comment]
I excluded worms though, I suggested for vertebrates and flys?

**Comment by balhoff** [comment]
> Yes, but here the whole organism lives inside something. I don't think it would apply to gene-specific annotation would it? The fish swims into the host, but swimming isn't the property of any specific gene product. Its a property of an entire organism.

It seems to me we have many biological processes that are not the properties of single gene products. For example, [swimming](http://purl.obolibrary.org/obo/GO_0036268). 😄 Sorry to be argumentative, just trying to keep things general since GO is not used only for gene annotation.

**Comment by ValWood** [comment]
> Sorry to be argumentative

Not at all!

I think swimming is OK, as a self-propelled movement , (genes control movement).
But I always have a bit of a problem with "behaviour terms" (we  have swimming behaviour too, which I find slightly odd) 

I was trying to find existing discussions of behaviour and could only find this:
https://github.com/geneontology/go-ontology/issues/18807
Of course, behaviour is controlled by genes but I struggle how most genes do not contribute to behaviour at some level  (it seems to be more of an outcome than a process).

> GO is not used only for gene annotation

GO might have uses outside of gene annotation, but its purpose is gene annotation.  Presumably  we only include processes that are in scope for gene annotation regardless of other uses? (we have enough difficultly with this  domain without extending  outside our required use)

I'd be interested to hear @cmungall  opinion.
But I'm happy to relieve the parasitic fishes of a taxon restriction if this is really considered to be in scope for GO.

**Comment by ValWood** [comment]
@pgaudet  I think it is OK to add
"never in vertebrates" if you agree I'll do  with a bunch of taxon constraints tomorrow

**Comment by pgaudet** [comment]
I am not sure why this is a problem. As you point out there are 1780 vertebrate annotations. People have annotated 'hijacked' proteins there. Until we have an alternative solution, I suggest we leave these annotations. 

Also - this is 'contained', being under interspecies interactions.

**Comment by ValWood** [comment]
I thought symbiont terms should only be used for symbiont proteins?
There are 400 human annotations to this term, this seems wrong? It isn't the evolved role?

But yes close if that's the correct way to capture.

**Comment by ValWood** [comment]
I see you say "unitl altertnative solution" so closing
